index.js

```diff
@@ -202,6 +202,9 @@ module.exports = {
         ],
         'no-useless-constructor': 'off',
         '@typescript-eslint/no-useless-constructor': 'warn',
+
+        // Linting configuration below this line is NOT taken from react-scripts
```

Comment:
Do we just want to move this above the other configuration as we then don't need to add this comment?